Source/santasyncservice/SNTSyncStage.mm

```diff
@@ -225,11 +225,7 @@ - (NSError *)performRequest:(NSURLRequest *)request
     return error;
   }
   if (data.length == 0) {
-    NSString *errStr = @"Response is empty";
-    SLOGE(@"%@", errStr);
-    return [NSError errorWithDomain:@"com.google.santa.syncservice"
-                               code:4
-                           userInfo:@{NSLocalizedDescriptionKey : errStr}];
+    return nil;
```

Comment:
nit: `if (data.length == 0) return nil;`

Reply:
I'm anti this!  Let's debate offline :)